### PR TITLE
Update npm dependencies and add LICENSE file

### PR DESCRIPTION
**Pull request by spadgett**
/assign @jwforres 
/cc @TheRealJon

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +22,8 @@
     "grunt-contrib-less": "^1.2.0",
     "grunt-contrib-watch": "^0.6.1",
     "grunt-wiredep": "~2.0.0",
-    "less-plugin-autoprefix": "^1.5.1"
+    "less-plugin-autoprefix": "^1.5.1",
+    "patternfly": ">=3.5.1 <4.0"
```
Comment (jwforres) [inline package.json:26]: When did we end up with a dependency on patternfly in this project?

Comment (spadgett) [reply package.json:26]: Looks like it's a dev dependency only, so not required. Is it just for testing with patternfly styles?

Comment (spadgett) [reply package.json:26]: I copied this from the bower.json dev dependencies

**Comment by spadgett** [comment]
I added an .npmignore file and updated the name to kubernetes-label-selector to match bower.json

**Review by jwforres — APPROVED** [review]

**Pull request merged**